anaSBitReadout.py

```diff
@@ -241,9 +245,9 @@
     ieta_h_sbitSize = ndict()
     ieta_h_delay = ndict()
 
-    for ieta in range(1, 9):
-        ieta_h_strip[ieta] = r.TH1F("h_ieta{0}_strips_vs_hit".format(ieta), "i#eta = {0} | i#phi (1,2,3)".format(ieta), 384, 0., 384.)
-        ieta_h_ch[ieta] = r.TH1F("h_ieta{0}_chan_vs_hit".format(ieta), "i#eta = {0} | i#phi (1,2,3)".format(ieta), 384, 0., 384.)
+    for ieta in range(1, maxiEta+1):
+        ieta_h_strip[ieta] = r.TH1F("h_ieta{0}_strips_vs_hit".format(ieta), "i#eta = {0} | i#phi (1,2,3)".format(ieta), 128*maxiPhi, 0., 128.*maxiPhi)
```

Comment:
I'm putting this variable in `mapping/ChamberInfo.py` for now, I'll implement that now. This can also be done for strips, ie a STRIPS_PER_IPHI variable if we want

Reply:
I think you can safely assume that here `128` should always be `CHANNELS_PER_VFAT`, as the partitions are almost surely defined by the locations of the VFATs

Reply:
With GE2/1 the eta mapping would be different, as each VFAT will control 64 strips in one eta partition and 64 strips in another, adjacent eta partition in a way, that a trigger pad will consist of 2 strips with the same phi and different eta coordinates. Thus said, the number of channels per VFAT will stay of course the same.

Reply:
To make sure I understand how we want to implement this: because we are going to have an equal number of strips to vfats, we should just replace all instances of 128 with CHANNELS_PER_VFAT since the distinction won't matter much, eg:
```
ieta_h_strip[ieta] = r.TH1F("h_ieta{0}_strips_vs_hit".format(ieta), "i#eta = {0} | i#phi (1,2,3)".format(ieta), 128*maxiPhi, 0., 128.*maxiPhi)
...
ieta_h_strip[eta].Fill((phi-1)*128+strip[0])
```

If this is correct, I'll go ahead and make that change

Reply:
Followed instructions from meeting